apps/webpack.js

```diff
@@ -95,15 +98,12 @@ if (envConstants.HOT) {
   });
 }
 
-if (process.env.CI) {
-  baseConfig.progress = false;
-}
-
 // modify baseConfig's preLoaders if looking for code coverage info
 if (envConstants.COVERAGE) {
-  baseConfig.module.preLoaders = [
+  baseConfig.module.rules.splice(-1, 1,
```

Comment:
This is a little action-at-a-distance-y - I think it's likely to break in the future when someone adds a new rule to the main config and forgets to change this index.  Can we do something that more specifically looks up the rule we want to replace?